Makefile

```diff
@@ -23,6 +23,32 @@ PUSH_COMMAND = IMAGE_TAG=${IMAGE_TAG} .scripts/travis/push-image.sh
 
 DC_BUILD = IMAGE_TAG=${IMAGE_TAG} docker-compose -f docker-compose.build.yml
 
+###########################
+#
+# Local Dev Environment
+#
+###########################
+
+start_networks:
+	-docker network create infra_postgres
+	-docker network create infra_api
+	-docker network create infra_rabbit
+
+start_infra: start_networks
+	# don't forget this just starts the containers, it doesn't assert that a particular
+	# thing is running
+	docker-compose -f docker-compose.infra.yml up -d
+	sleep 10
```

Comment:
This is "ok" for now - raising a ticket to implement resilience in the event bus library.